pkg/logging/logfields/logfields.go

```diff
@@ -56,6 +56,12 @@ const (
 	// IPAddr is an IPV4 or IPv6 address
 	IPAddr = "ipAddr"
 
+	// V4HealthIP is an address used to contact the cilium-health endpoint
+	V4HealthIP = "v4healthIP"
```

Comment:
How about `healthIP.IPv4` and `healthIP.IPv6` to be consistent with e.g. `policyID.L3` above?